### PR TITLE
LPS-187750 filter search results with repositories

### PR DESCRIPTION
**Pull request by marcogalluzzi**
See this story for requirements https://liferay.atlassian.net/browse/LPS-177661 
NOTE: buttons specified in requirements only refer to switch between Repositories but previous implementation also has the current Folder button, which I didn't remove.

I'm not sure about method `_isExternalRepositorySearch()` created in last commit in class `DLAdminDisplayContext`. Please give advise. Thanks

Frontend changes to review: file `search_info.jsp`

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
The PR can be reviewed, but I'm discussing with the designer. Maybe some requirement is missing or not correctly implemented. So maybe some other commit is needed. This is why is ON HOLD

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp [hunk 1]
```diff
@@ -68,7 +66,6 @@ entriesChecker.setRememberCheckBoxStateURLRegex("^(?!.*" + liferayPortletRespons
 
 	<div class="document-container" id="<portlet:namespace />entriesContainer">
 		<liferay-ui:search-container
-			emptyResultsMessage='<%= LanguageUtil.format(request, "no-documents-were-found-that-matched-the-keywords-x", HtmlUtil.escape(keywords), false) %>'
```
Comment (ambrinchaudhary) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:71]: Is this correct? Do we really want to remove the empty message?

**Review by ambrinchaudhary — APPROVED** [review]
Just a minor question, otherwise frontend changes LGTM

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp (hunk 1 above)
Comment (marcogalluzzi) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:71]: The empty message is removed here in the JSP, but is added during the creation of the `searchContainer` in JAVA
Reason: more logic is needed to decide which message to show 😄

**Comment by marcogalluzzi** [comment]
ci:test:sf

**Comment by marcogalluzzi** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:71]: Thanks for the clarification, I didn't review the JAVA code 🙈

**Comment by ambrinchaudhary** [comment]
> Hi @ambrinchaudhary can you please check if the new location of the clay:alert is correct? thanks

Do you mean visually?? Then I will download your PR and test it locally (for next pulls, it is very useful to attach a screenshot 😉 )

**Comment by marcogalluzzi** [comment]
> > Hi @ambrinchaudhary can you please check if the new location of the clay:alert is correct? thanks
> 
> Do you mean visually?? Then I will download your PR and test it locally (for next pulls, it is very useful to attach a screenshot 😉 )

Hi @ambrinchaudhary I meant: is it the tag located inside the correct section of the jsp, visually I can tell you it looks fine 👍

**Comment by marcogalluzzi** [comment]
ci:stop

**Comment by marcogalluzzi** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
> Hi @ambrinchaudhary I meant: is it the tag located inside the correct section of the jsp? visually I can tell you it looks fine 👍 (no need to download the PR)

We do not have a particular section for alerts, it is only about if visually is ok. I see it is inside the _sidenav-content_ (so if you open the info panel it is reduced as well as the content), so for me yes, it is placed in the correct side.

**Comment by marcogalluzzi** [comment]
Hi @WYZ-Issac I haven't changed the kernel package so this error should be in upstream right? Do I have to fix it?

```
[exec]   PACKAGE_NAME                                       DELTA      CUR_VER    BASE_VER   REC_VER    WARNINGS  
     [exec] = ================================================== ========== ========== ========== ========== ==========
     [exec] * com.liferay.portal.kernel.upload                   MINOR      10.0.0     10.0.0     10.1.0     VERSION INCREASE REQUIRED
     [exec]   <   interface  com.liferay.portal.kernel.upload.FileItem
     [exec]     +   method     getTempFile()
     [exec]       +   access     abstract
     [exec]       +   return     java.io.File
     [exec] [Baseline Report] Mode: diff (persisted)
     [exec] Semantic versioning is incorrect while checking /opt/dev/projects/github/liferay-portal/portal-kernel/portal-kernel.jar against /opt/dev/projects/github/liferay-portal/.gradle/caches/modules-2/files-2.1/com.liferay.portal/com.liferay.portal.kernel/114.0.0/66f8b6230e5309aa50dc662c7cf17779de773eb7/com.liferay.portal.kernel-114.0.0.jar
```

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward:force

**Comment by adolfopa** [comment]
@marcogalluzzi Errors look unrelated; forcing the forward.

**Pull request closed without merging**